packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -74,6 +74,18 @@ endpoint_tests:
     type:
       - master
   - tests:
+      # This test checks whether AR's `/service` endpoint is able to route to
+      # the correct service even though the request url contains duplicate
+      # slashes.  Duplicate slashes is AFAIK not a valid RFC behaviour, but
```

Comment:
Attempting to @jgehrcke with the above: Let's be precise here.

In particular if this is behaviour not supported by other systems or standards, this is exactly the kind of test that might break in a big rewrite. We want the maintainer at that point to be able to make an informed decision on whether or not to keep the test.

That decision may be based on:
* To what extent do people rely on this functionality - we may never know
* Is this required behaviour to meet standards we aim or claim to meet
* Is this required behaviour to match behaviour of other tools that we aim to match

This comment can try to address some of this.

> To what extent do people rely on this functionality

Why are we adding this test? We add tests in order to provide regression-safety for functionality we do not want to accidentally disappear. Why is this functionality in this category? Is it because someone requested it?

> Is this required behaviour to meet standards we aim or claim to meet

I'm not sure but perhaps that RFC will give us a clue.
Actually the double slash, from what I can gather should *not* be considered the same as a single slash. In fact a double slash should indicate an empty path.
See https://stackoverflow.com/a/40683096.

> Is this required behaviour to match behaviour of other tools that we aim to match

Maybe?

I see

>  in (most?) modern operating systems (Linux/Unix, Windows), multiple path separators in a row do not have any special meaning, so /path/to/foo and /path//to////foo would eventually map to the same file. - https://stackoverflow.com/a/10161264

and this adds to the idea that this is behaviour we should support.